### DISCUSSION TITLE
EQ3 with 2 Tasmota devices

### DISCUSSION THREAD

**Discussion by dog-rgb** [arendst/Tasmota#14099] [Q&A] [answered]
Hi,

the EQ3 reports in dedicated object trees which are independent to the "topic" parameter. Usually the data will be written in objects which have the "topic" included in the path. What happens if I have two Tasmota Bluetooth devices in range of one dedicated EQ3?

Will both write into the same EQ3 object in the iobroker? If yes, will that be a problem? If not, any possibility to avoid it?

Would it be better if the EQ3 object path would include the "topic" as it is done for other objects?

**Comment by barbudor** [comment, marked as answer]
This is the choice of the driver's writer.
I'm not using the EQ3 driver but could you use BLEAlias to give a different alias to the EQ3 on the 2 different Tasmota so you caneasiliy ignore one ?

**Reply by dog-rgb** [reply]
Thank you for the proposal, it works.

But it would be great if the writer of the driver reads this and think of using the "topic" parameter also for the stat report (according to the docu the "topic" is used for commands).
